common/kokkos-sampler/kp_sampler_skip.cpp

```diff
@@ -82,9 +125,7 @@ void kokkosp_init_library(const int loadSeq, const uint64_t interfaceVer,
       printf("KokkosP: Next library to call: %s\n", nextLibrary);
       printf("KokkosP: Loading child library ..\n");
     }
-
```

Comment:
What's up with all these whitespace changes?

Reply:
I will fix, making sure the code file is indeed getting processed through clang-format.

Reply:
@masterleinad  Does the new file committed properly address the issues you have with whitespace changes?